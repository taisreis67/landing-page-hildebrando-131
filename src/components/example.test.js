import React from "react"
import renderer from "react-test-renderer"
import SectionHeader from "./section-header"

describe("SectionHeader", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SectionHeader title="Chuchu" description="chuchu chu chuchu" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})