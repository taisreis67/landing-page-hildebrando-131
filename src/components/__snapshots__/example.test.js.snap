// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader renders correctly 1`] = `
Array [
  <h2>
    Chuchu
  </h2>,
  <p
    style={
      Object {
        "color": "#B0B8BA",
      }
    }
  >
    chuchu chu chuchu
  </p>,
]
`;
